PTBR/SurvivalGuide_PTBR.txt

```diff
@@ -1,101 +1,101 @@
-                                    " <H1> DASHBOARD <BR> <TEXT> <IMAGECENTRE:media/ui/tutimages/cardashboard.png> <LINE> On a vehicle's Dashboard you can see helpful read-outs of information like the fuel levels, speed and RPM. <BR> Icons on top of the dash will also tell you engine status, whether the trunk is open, whether the headlights are operational, and so on. <BR> " ..
-                                    " <H1> RADIAL MENU <BR> <TEXT> If you're inside or next to a vehicle press <SPACE><KEY:VehicleRadialMenu>. This will open its radial menu. <BR> <IMAGECENTRE:media/ui/tutimages/carradialmenu.png> <LINE> From here you can switch seats, lock doors, operate lights, open up the Mechanics window, and more. <BR> " ..
-                                    " <H1> MECHANICS WINDOW <BR> <TEXT><IMAGECENTRE:media/ui/tutimages/carmechanics.png> <LINE> An unskilled survivor can check if their battery is still okay, see fuel levels, and notice whether certain parts of the car appear damaged. A survivor with better Mechanic skills will be able to see extra details, and install and uninstall parts. <BR> For more information on this and the Mechanic skill, click the info icon in the top left corner of the Mechanic window. <BR>",
+    SurvivalGuide_entrie9title = "#9 - VEÍCULOS",
+    SurvivalGuide_entrie9txt = " <CENTRE> "<SIZE:large> #9 - VEÍCULOS <LINE> <IMAGECENTRE:media/ui/spiffo/driver.png> <LINE> <SIZE:medium> Os veículos trazem benefícios de sobrevivência, mas também trazem ruído! Um motor acelerado atrairá a horda de uma longa distância...",
+    SurvivalGuide_entrie9moreinfo = " <H1> PULE DENTRO <BR> <IMAGECENTRE:media/ui/tutimages/spiffovehicle.png> <LINE> <TEXT> Se você tiver sorte, um dono de carro louco pode ter deixado suas chaves na porta, na ignição ou no porta-luvas. Alternativamente, procure zeds próximos, caso um deles seja o proprietário. Da mesma forma, se o veículo escolhido estiver estacionado do lado de fora de uma casa, faça uma busca. A chave pode estar dentro. <BR> Ainda sem alegria? Por que não fazer um pequeno roubo de carro? Esmague a janela do veículo para destravá-lo. Se você tiver habilidades mecânicas e elétricas altas o suficiente, poderá fazer ligação direta. <BR> " ..
+                                    " <H1> HORA DE DIRIGIR <BR> <TEXT> Clique na ignição para inserir a chave e, em seguida, clique no ícone do motor no painel (ou pressione N) para ligar o motor. Alternativamente, basta pressionar a tecla avançar <SPACE><KEY:Forward><SPACE> para rolar. <BR> Se a condição do motor do carro não for alta, ele pode não dar partida na primeira vez. Se estiver se recusando a ligar, você pode estar completamente sem gasolina e talvez seja hora de inspecionar a janela Mecânica... <BR> Dirija com cautela! Quanto mais alto um carro, maior o raio de zumbis próximos que serão alertados e começarão a seguir o barulho...  <BR> " ..
```

Comment:
Clique na ignição para inserir a chave e, em seguida, clique no ícone do motor no painel (ou pressione N) para ligar o motor. Alternativamente, basta pressionar a tecla avançar ... para **acelerar.** ... Se a condição do motor do carro não for alta, ele pode não dar partida na primeira vez. Se estiver se recusando a ligar, você pode estar completamente sem **combustível** e talvez seja hora de **verificar a Mecânica do Veículo**... Dirija com cautela! Quanto mais **barulhento for** um carro, maior o raio de zumbis próximos que serão alertados e começarão a seguir o barulho...